jeongjaein/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/MainActivity.kt

```diff
@@ -0,0 +1,38 @@
+package com.exam.elevenstreet
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.exam.elevenstreet.R.layout.activity_main
+import com.example.elevenstreet.ProductResponse
+import com.example.elevenstreet.ProductXmlPullParserHandler
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    lateinit var productList: List<ProductResponse>
```

Comment:
```suggestion
    private lateinit var productList: List<ProductResponse>
```
다른클래스에서 접근할 변수가 아니니  private 으로 선언해주세요.

Reply:
알겠슴다!